lang/en.json

```diff
@@ -107,6 +111,7 @@
     "cat.ssl.results.champion": "Champions!",
     "cat.ssl.r": "Results",
 
+    "cat.ssl.robocup22": "1st Div B<br>1st Ball placement technical challenge<br>2nd Vision Blackout<br>3rd Dribbling challenge",
```

Comment:
I think it would be better to use the initial letters in uppercase, something like `Vision Blackout Challenge` and `Ball Placement Challenge`